### ISSUE TITLE
can not find snopt folder

### ISSUE THREAD

**Issue by MiaoLi** [RobotLocomotion/drake-distro#17]
Hi 
I try to install drake using: 
git clone https://github.com/RobotLocomotion/drake-distro.git --recursive -b precompiled
But it gives error:
===================
remote: Repository not found.
fatal: repository 'https://github.com/RobotLocomotion/snopt/' not found
Clone of 'http://github.com/RobotLocomotion/snopt' into submodule path 'snopt' failed
==================
Should I use some other more recent branches? 
Thanks.

**Comment by RussTedrake** [comment]
Yes. I'm allowed to give away the compiled snopt, but not the source.  (so precompiled has it included, but it requires permissions).

I'd recommend the rigidbody distro.
